core/templates/dev/head/profile/preferences.html

```diff
@@ -77,6 +77,28 @@ <h2 class="oppia-page-heading">Preferences</h2>
 
         <div role="form" class="form-horizontal">
           <div class="form-group">
+            <label class="col-lg-2 col-md-2 col-sm-2">Subject Interests</label>
+            <div class="col-lg-10 col-md-10 col-sm-10">
+              <div ng-if="hasPageLoaded">
+                <select2-dropdown
+                  item="$parent.subjectInterests"
+                  tag-mode="true" allow-multiple-choices="true"
+                  invalid-search-term-message="Add a new subject interest (using lowercase letters and spaces)..."
+                  new-choice-regex="<[TAG_REGEX]>" width="100%"
```

Comment:
I see this when I inspect element:
textAngular-rangy.min.js:10 'Range.detach' is now a no-op, as per DOM (http://dom.spec.whatwg.org/#dom-range-detach).

I'm not sure what it means when new-choice-refex is not being sent in properly. My syntax matches other instances were new-choice-regex is used.